diff -ruN deepgram_orig/clients/agent/v1/websocket/options.py deepgram/clients/agent/v1/websocket/options.py
--- deepgram_orig/clients/agent/v1/websocket/options.py	2025-05-11 16:44:31.042456006 +0300
+++ deepgram/clients/agent/v1/websocket/options.py	2025-05-11 16:44:24.388487702 +0300
@@ -241,7 +241,7 @@
     This class defines any configuration settings for the context.
     """
 
-    messages: Optional[List[Tuple[str, str]]] = field(
+    messages: Optional[List[dict[str, str]]] = field(
         default=None, metadata=dataclass_config(exclude=lambda f: f is None)
     )
     replay: Optional[bool] = field(default=False)
diff -ruN deepgram_orig/___PATCHED___.txt deepgram/___PATCHED___.txt
--- deepgram_orig/___PATCHED___.txt	1970-01-01 03:00:00.000000000 +0300
+++ deepgram/___PATCHED___.txt	2025-05-11 16:44:31.281454902 +0300
@@ -0,0 +1 @@
+patched
